packages/react/src/components/popover/popover.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Alignment,
+  c,
+  classy,
+  m,
+  PopoverProps as BaseProps,
+  Position,
+} from '@onfido/castor';
+import React, { ReactNode, RefObject, useRef, useState } from 'react';
+import { createPortal } from 'react-dom';
```

Comment:
Probably this PR should now include [`react-dom` as dependency](https://github.com/onfido/castor/pull/752/files#diff-1f344ac391eeecc21ec0f01fb07430a47f4b80d20485c125447d54c33c4bbfc4R33)?

Reply:
Ah yes, thanks for the reminder.